documentation/error-handling.md

```diff
@@ -0,0 +1,64 @@
+# Error handling
+
+UI extensions support standard web errors. You can add an `unhandledrejection` listener for unhandled promise rejections or an `error` listener for other exceptions like JavaScript runtime errors or resources that fail to load.
```

Comment:
I forget, did we actually implement these for the native JS sandbox as well?

Reply:
Can we add a quick recommendation here? Something like, "If you don’t already have an error reporting tool, but you want to be alerted when errors occur in your extension, we recommend X. It has a free version, and has the smallest bundle size of the error tracking services documented in this guide."

Reply:
> did we actually implement these for the native JS sandbox as well?

By `native JS sandbox`, you mean mobile, don't you? If so, no. Current Android gets this for free because it switch to use WebView like Web. I haven't verified it in current iOS as it is using JavascriptCore. 

For future Android and iOS powered by React Native in Mobile Shopify and POS, we get it for free as they will be both using WebView (hidden) and WebWorker like Web. So, I wou

Reply:
I was talking about mobile, yeah. Thanks for the explanation!